uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -185,8 +188,10 @@ define([
             if (checkError(data)) {
               deferred.reject(data.error_message);
             }
-            //returns array of single object hence getting first object
-            deferred.resolve(data.values[0]);
+            else {
```

Comment:
Both styles are being used
```if() {
}
else {
}
```
and 
```if() {
} else {
}
```
Consistency should be kept.
I prefer
```if() {
} else {
}
```
Please check all instances in this file